source/common/config/config_provider_impl.h

```diff
@@ -219,18 +219,21 @@ class ConfigSubscriptionCommonBase
   using ConfigUpdateCb =
       std::function<ConfigProvider::ConfigConstSharedPtr(ConfigProvider::ConfigConstSharedPtr)>;
   void applyConfigUpdate(
-      const ConfigUpdateFn& update_fn, const Event::PostCb& complete_cb = []() {}) {
+      const ConfigUpdateCb& update_fn, const Event::PostCb& complete_cb = []() {}) {
     // It is safe to call shared_from_this here as this is in main thread, and destruction of a
     // ConfigSubscriptionCommonBase owner (i.e., a provider) happens in main thread as well.
     auto shared_this = shared_from_this();
     tls_->runOnAllThreads(
         [this, update_fn]() {
           tls_->getTyped<ThreadLocalConfig>().config_ = update_fn(this->getConfig());
         },
-        /*During the update propagation, a subscription may get teared down in main thread due to
+        /* During the update propagation, a subscription may get teared down in main thread due to
```

Comment:
Tiny nit: please use `//` comments here for the block comment.

Other than these tiny nits, LGTM.

Reply:
done

Reply:
thx!